### PR TITLE
Merge qa-0.9 with relase branch

### PR DESCRIPTION
**Pull request by OlleLarsson**
**What this PR does / why we need it**:

**Special notes for reviewer**:

**Checklist:**

- [ ] Added relevant notes to [WIP-CHANGELOG.md](https://github.com/elastisys/compliantkubernetes-apps/blob/main/WIP-CHANGELOG.md)
- [ ] Proper commit message prefix on all commits
- [ ] Updated the [public facing documentation](https://github.com/elastisys/compliantkubernetes)
- Is this changeset backwards compatible for existing clusters? Applying:
    - [ ] is completely transparent, will not impact the workload in any way.
    - [ ] requires running a migration script.
    - [ ] will create noticeable cluster degradation.
          E.g. logs or metrics are not being collected or Kubernetes API server
          will not be responding while upgrading.
    - [ ] requires draining and/or replacing nodes.
    - [ ] will change any APIs.
          E.g. removes or changes any CK8S config options or Kubernetes APIs.
    - [ ] will break the cluster.
          I.e. full cluster migration is required.

### CODE REVIEW

**Review by lentzi90 — APPROVED** [review]
Looks good! Have not checked all the code, but looked through the commits and they seemed correct. :slightly_smiling_face:

File: WIP-CHANGELOG.md [hunk 1]
```diff
@@ -1,91 +1,74 @@
 ### Release notes
```
Comment (Xartos) [inline WIP-CHANGELOG.md:1]: Shouldn't this file be empty?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,10 +1,87 @@
...
 <!-- BEGIN TOC -->
+- [v0.8.0](#v080---2020-12-11)
 - [v0.7.0](#v070---2020-11-09)
 - [v0.6.0](#v060---2020-10-16)
 - [v0.5.0](#v050---2020-08-06)
 <!-- END TOC -->
 
+-------------------------------------------------
+
+## v0.8.0 - 2020-12-11
```
Comment (Xartos) [inline CHANGELOG.md:11]: Shouldn't this be the changelog for 0.9.0?

File: WIP-CHANGELOG.md (hunk 1 above)
Comment (OlleLarsson) [reply WIP-CHANGELOG.md:1]: Is this perhaps done by the script mentioned in step 3?
https://github.com/elastisys/compliantkubernetes-apps/tree/main/release#major-and-minor-releases

Comment (OlleLarsson) [reply WIP-CHANGELOG.md]: Fixed by running script.

File: CHANGELOG.md (hunk 2 above)
Comment (OlleLarsson) [reply CHANGELOG.md:90]: Fixed.

**Pull request merged**